scripts/fetch.js

```diff
@@ -0,0 +1,235 @@
+        request({
+            url: locality.url,
+            encoding: null
+        }, function (error, response, body) {
+            if (!error && response.statusCode === 200) {
+                fs.writeFileSync(path.resolve('bikelanes', 'temp.zip'), body);
+                console.log('Converting Shapefile to GeoJson for ' + locality.name);
+                var formData = { upload: fs.createReadStream(path.resolve('bikelanes', 'temp.zip')), targetSrs: 'EPSG:4326' };
+                request.post({
+                    url: 'http://ogre.adc4gis.com/convert',
+                    formData: formData
+                }, function (error, response, body) {
+                    fs.unlinkSync(path.resolve('bikelanes', 'temp.zip'));
+                    if (!error && response.statusCode === 200) {
+                        mapFilterSave(JSON.parse(body), locality);
+                    } else {
+                        console.log('ERROR: ' + error);
+                    }
+                });
+            } else {
+                console.log('ERROR: ' + error);
+            }
+        })        
+    }
+    else {
+        request({
+            url: locality.url,
+            json: true
+        }, function (error, response, body) {
+            if (!error && response.statusCode === 200) {
+                mapFilterSave(body, locality);
+            } else {
+                console.log('ERROR: ' + error);
+            }
+        })
+    }
+    
+}
+
+function mapFilterSave(geoJson, locality) {
+    //each locality has their own labeling & categorizing system.  Normalize them.
+    console.log('mapping raw file for ' + locality.name);
+    console.log('# of raw features: ' + geoJson.features.length);
+    var mappedGeoJson = locality.mappingFunction(geoJson);
+
+    //filter it so we don't have sharrows and wide shoulders and other things WABA doesn't consider infrastructure
+    console.log('filtering mapped file for ' + locality.name);
+    mappedGeoJson.features = mappedGeoJson.features.filter(isRealBikeFacility);
+
+    console.log('writing finalized file to disk for ' + locality.name);
+    //write to disk
+    fs.writeFileSync(path.resolve('bikelanes', locality.filename), JSON.stringify(mappedGeoJson));
+    locality.done = true;
+    if (locality.onDone) locality.onDone();
+}
+
+function mocomap(rawGeoJson) {
+    //keep objectid
+    //keep name
+    //convert category = 'Paved Off-Road Trail' to wabaclassification = 'Paved Trail'
+    //convert category = 'Bike Lanes' to wabaclassification = 'Bike Lane'
+    //convert category = 'Bike-Friendly Shoulders' to wabaclassification = 'Shoulder'
+    //convert category = 'Signed On-Road Route' to wabaclassification = 'Signed Route'
+    //convert category = 'Natural Surface' to wabaclassification = 'Unpaved Trail'
+    //convert category = 'Separated Bike Lanes' to wabaclassification = 'Separated Bike Lanes'
+
+    var geojson = { "type": "FeatureCollection", "features": [] };
```

Comment:
Any reason this needs to be double quotes? This is so that it becomes proper GeoJSON, right? If so, I may end up re-converting this project from single quotes to double quotes again 😓 .